go-controller/pkg/clustermanager/egressip_controller_test.go

```diff
@@ -0,0 +1,2349 @@
+package clustermanager
+
```

Comment:
what tests in this file are new and not copied over?

Reply:
All of them were copied over. Nothing new in this file. New UTs were added only in `egressip_test.go` in `ovn` pkg to reflect the new its for LRSR and the way we add and remove things in NBDB.